Code-Hills/codehills-dev-profile-fe
# Feature: real time user search

# Real-time user search by display name

## Description
This pull request adds a feature that will allow the admin to search among the users' list

## Screenshots
![image](https://github.com/Code-Hills/codehills-dev-profile-fe/assets/52130946/8072d031-151f-417e-a39e-ab58bcb1f710)


## Testing
The following tests were added to ensure the new admin functionality works as expected:
- checkout to the `ft-search-user` branch
- start the app by `yarn dev`
- visit the page in the browser at localhost:[port]/dashboard ad the chose users tab

## Known Issues
There are currently no known issues with this pull request.
